fractions/microprofile/microprofile/pom.xml

```diff
@@ -54,6 +54,13 @@
       <groupId>org.wildfly.swarm</groupId>
       <artifactId>jaxrs-jaxb</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Shouldn't this fraction also have dependencies on the new spec fractions?

Reply:
Yes